include/cantera/thermo/PengRobinson.h

```diff
@@ -173,6 +173,9 @@ class PengRobinson : public MixtureFugacityTP
     void setBinaryCoeffs(const string& species_i, const string& species_j, double a);
     //! @}
 
+    //! Return common parameters used by the Peng-Robinson equation of state
+    AnyMap getEosParameters() override;
```

Comment:
For the derived class, the docstring should what parameters are returned, and what the key names are in the `AnyMap`.